### PR TITLE
docs: remove locale from route name

### PR DESCRIPTION
**Pull request by johnleider**
## Motivation and Context
Remove the locale from the generated route name

### CODE REVIEW

File: packages/docs/src/components/getting-started/WireframeExamples.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      :key="i"
+      cols="12"
+      md="6"
+      lg="4"
+    >
+      <router-link
+        :to="{
+          name: `wireframes-${wireframe}`
+        }"
+        class="text--primary text-decoration-none"
```
Comment (KaelWD) [inline packages/docs/src/components/getting-started/WireframeExamples.vue:14]: text--primary doesn't exist

File: packages/docs/vite.config.ts [hunk 2]
```diff
@@ -87,31 +87,48 @@ export default defineConfig(({ command, mode }) => {
...
-        pagesDir: [
-          { dir: 'src/pages', baseRoute: '' },
+        dirs: [
+          { dir: 'src/pages', baseRoute: 'pages' },
           { dir: 'src/api', baseRoute: 'api' },
+          { dir: 'src/wireframes', baseRoute: 'wireframes' },
         ],
         extendRoute (route) {
-          if (['index', 'all'].includes(route.name)) {
+          if (['index', 'all'].includes(route.component)) {
```
Comment (KaelWD) [inline packages/docs/vite.config.ts:96]: This never matches, and probably didn't before either.

**Comment by KaelWD** [comment]
Wireframes were at `examples/wireframes/:wireframe/` before

File: packages/docs/vite.config.ts (hunk 2 above)
Comment (johnleider) [reply packages/docs/vite.config.ts:96]: double check. It didn't work with route.name, but route.component uses the actual index.md filename.

Comment (KaelWD) [reply packages/docs/vite.config.ts:96]: yeah but it's never just `'index'` or `'all'`

**Pull request merged**